src/pages/dashboard/adduser/index.tsx

```diff
@@ -8,10 +8,9 @@ import InvitationMail from "@/ui/forms/invitationMail";
 import SideMenu from "@/ui/menu/sidemenu";
 import Page from "@/ui/page";
 
+
+
```

Comment:
the folder names should be in kebab case, check this doc => https://www.freecodecamp.org/news/programming-naming-conventions-explained/#what-is-kebab-case